### PR TITLE
Add Oopy detection

### PR DESCRIPTION
**Pull request by ysm0622**
website:
- https://oopy.us/
- https://oopy.io/

examples:
- https://tosspayments-dev.oopy.io/
- https://jobs.class101.net/
- https://www.opgg.team/
- https://careers.amazevr.com/
- https://watcha.team/
- https://woowahan.oopy.io/
-

### CODE REVIEW

File: src/technologies/o.json [hunk 1]
```diff
@@ -716,6 +716,27 @@
     "scriptSrc": "\\.speedtestcustom\\.com/",
     "website": "https://www.ookla.com/speedtest-custom"
   },
+  "oopy": {
+    "cats": [
+      51
+    ],
+    "description": "Oopy provides you with a simple and easy way to turn your Notion page into a performant website.",
+    "icon": "oopy.png",
```
Comment (nurbek91) [inline src/technologies/o.json:724]: 32x32
Oopy

File: src/technologies/o.json [hunk 2]
```diff
@@ -716,6 +716,27 @@
     "scriptSrc": "\\.speedtestcustom\\.com/",
     "website": "https://www.ookla.com/speedtest-custom"
   },
+  "oopy": {
```
Comment (nurbek91) [inline src/technologies/o.json:719]: Let's write the names of technologies with a capital letter, just like in the descriptions

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for your PR!!!
Requires cosmetic updates to bring to the ideal

File: src/technologies/o.json (hunk 2 above)
Comment (ysm0622) [reply src/technologies/o.json]: [609b56c](https://github.com/wappalyzer/wappalyzer/pull/6720/commits/609b56c9c6896a03b4dbdf5b7f8e4ccf9fdf442f)

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**